tests/IntegrationTest.php

```diff
@@ -45,4 +46,32 @@ public function testEvents()
             $statistics[0]
         );
     }
+
+    public function testSince()
+    {
+        $now = CarbonImmutable::now();
+        $yesterday = $now->subDay();
+        Carbon::setTestNow($yesterday);
```

Comment:
I don't think this does what you think it does. This is just running the whole test yesterday.

If you do this:
```php
        $now = Carbon::now();
        Carbon::setTestNow($now);
        $now->subDay()
```

You can advance time later simply with `$now->addDay();'.

Reply:
I think we ended up in a reasonable state with the latest changes.